addons/website/controllers/binary.py

```diff
@@ -5,7 +5,7 @@
 
 class WebsiteBinary(Binary):
     @http.route([
-        '/web/assets/<int:website_id>/<unique>/<string:filename>'], type='http', auth="public")
```

Comment:
That one is technically not always readonly but apparently they do some magic to make it readonly when needed in the super call so this seems right.